templates/bundles/DemosPlanCoreBundle/DemosPlanProcedure/administration_send_email_confirm.html.twig

```diff
@@ -31,7 +31,7 @@
             {{ 'email.body'|trans }}
         </label>
 
-        <div>{{ templateVars.preparationMail.mailBody|wysiwyg }}</div>
+        <div class="email-body">{{ templateVars.preparationMail.mailBody|wysiwyg }}</div>
```

Comment:
I think this could also be solved by applying `c-styled-html`, which I believe we use for cases like this (when we get html from the BE)